src-electron/rest/file-ops.js

```diff
@@ -40,11 +41,15 @@ const studio = require('../ide-integration/studio-rest-api')
  */
 function httpPostFileOpen(db) {
   return async (req, res) => {
-    let { zapFilePath, ideProjectPath } = req.body
-    if (req.body.search?.includes('filePath=%2F')) {
-      ideProjectPath = dbEnum.sessionKey.ideProjectPath
-      zapFilePath = req.body.search?.split('filePath=')
-      zapFilePath = zapFilePath[1]?.replaceAll('%2F', '//').trim()
+    let { ideProjectPath } = req.body
+    let search = req.body.search
+    let zapFilePath
+    if (search[0] === '?') {
```

Comment:
Instead of manually checking for the ?, URLSearchParams() will automatically skip the ? if present.